### PR TITLE
Fix links in PR details view.

### LINKED ISSUE #1134: Links in Pull Request description not clickable.
In 2.3.0.16, links in PR details view are not clickable.

### PR DESCRIPTION
**Pull request by grokys**
The command wasn't being handled so links were disabled.

Fixes #1134

### CODE REVIEW

File: src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs [hunk 1]
```diff
@@ -306,5 +306,15 @@ void ViewCommentsClick(object sender, RoutedEventArgs e)
...
             catch { }
         }
+
+        void OpenHyperlink(object sender, ExecutedRoutedEventArgs e)
+        {
+            var url = e.Parameter.ToString();
+
+            if (!string.IsNullOrEmpty(url))
+            {
+                VisualStudioBrowser.OpenUrl(new Uri(url));
```
Comment (jcansdale) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: What would happen if `url` was malformed?

Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: As @shana said in https://github.com/github/VisualStudio/pull/1133#discussion_r133485930 - these links are created by us, so they shouldn't be malformed.

Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: Oh actually sorry, they're not created by us here. If they're malformed they'll be passed to the browser malformed, as they would on .com I imagine.

Comment (grokys) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: Actually thinking more about this, this would be worth checking...

Comment (jcansdale) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: I think `new Uri(...)` might throw.  Try `new Uri("boom://oops")`.

Comment (shana) [inline src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml.cs]: Probably good to use `Uri.TryParse()` on this one

**Pull request merged**